### ISSUE TITLE
Update script not work with way

### ISSUE THREAD

**Issue by thangdc94** [openmaptiles/openmaptiles#867]
I'm using your update script which documented [here ](https://github.com/openmaptiles/openmaptiles/blob/master/UPDATE.md) to update way in my project. 

In this testcase i moved 2 nodes of a way then created `changes.osc.gz` (diff of `new.osm` and `old.osm`), created `tiles.txt` and run `docker-compose run generate-changed-vectortiles`.

The left picture is `old.osm` and the right is `new.osm`

![image](https://user-images.githubusercontent.com/13900522/81913775-9e64f300-95fa-11ea-9fde-ee2e55806607.png)

And I got mbtiles like this. As you see, it looks like my `old.osm` file
![image](https://user-images.githubusercontent.com/13900522/81914803-f8b28380-95fb-11ea-82ba-61a35e7ccc45.png)

OMT Update works fine with POI and house number but seems not work with way layer and other layers.

I run `make generate-tiles` with `new.osm` file and mbtiles updated.

Did I do anything wrong?
What should I do to update some small parts of map?

**Comment by thangdc94** [comment]
After some tests and debugging I found out that the error occur with `make import-diff`. This tool did not generate any file in `diffdir`. It might be `imposm3` issues. 

I think `imposm3` only work with node in `*.osc` file because osc file doen't contains any infomation of old nodes of way. 

Is there any idea how to fix this?
Or maybe I need to submit issue to `imposm3`.

**Comment by nyurik** [comment]
Thanks for the report. Last time i checked the import-diff, it was working properly, but we definitely should build some integration tests for it to be certain. Ideally we should have some well defined base file plus a few updates saved, so that each run we could make against that data set rather than getting live data.

Note that at the moment we are using a very old version of `imposm`. The latest version could have fixed that issue - to try it, use `export TOOLS_VERSION=5.0-imp10-dev` before importing your data. Also, you may need to set `DIFF_MODE=true` for the imposm to set up correct caching.

**Comment by thangdc94** [comment]
@nyurik I tried your solution but it doesn't solve the problem. 
I set both `TOOLS_VERSION=5.0-imp10-dev` in `.env` and `DIFF_MODE=true`.

`imposm3` version 0.10.0  in `openmaptiles-tools:5.0-imp10-dev` seems doesn't solve the problem. Still no file in `diffdir` when I update 2 nodes of a way

This is what my changes.osc.gz look likes:

```
<?xml version='1.0' encoding='UTF-8'?>
<osmChange version="0.6" generator="Osmosis 0.47.4">
  <modify>
    <node id="6017710227" version="2" timestamp="2018-10-27T13:10:22Z" uid="1293386" user="Sal73x" changeset="63927815" lat="20.37049" lon="106.56634"/>
    <node id="6017710228" version="2" timestamp="2018-10-27T13:10:22Z" uid="1293386" user="Sal73x" changeset="63927815" lat="20.37626" lon="106.56051"/>
  </modify>
</osmChange>
```

I think OMT postgres DB imported by `imposm` should contains `way_id` and `way_node`. Do you know which tables contain these data?

**Comment by nyurik** [comment]
@thangdc94 did you set the `DIFF_MODE=true` before the initial import, or only when you ran the updater?

**Comment by thangdc94** [comment]
@nyurik I set `DIFF_MODE=true` before the initial import,

**Comment by nyurik** [comment]
Hmm.. I wonder what could be causing it.  I will try to investigate, but do let me know if you figure it out.

**Comment by thangdc94** [comment]
Yeah I will answer it here if I find the solution.
Does OMT tables contain `way_id` and `way_node`? It will help my debug

**Comment by nyurik** [comment]
i will have to rebuild to check -- use `make psql` to examine the tables.

**Comment by thangdc94** [comment]
@nyurik After some debug, I found out that my config in `.env` was wrong.
I set `DIFF_MODE=TRUE` instead of `DIFF_MODE=true`.

I read `import-osm` script which only accept variable DIFF_MODE with lowercase value of `true`.
https://github.com/openmaptiles/openmaptiles-tools/blob/bb072fee37/bin/import-osm#L22

After changing this variable, updating work fine.
Sorry for my mistakes.
I think I can close this issue here.
